.github/workflows/build-graphscope-wheels-macos.yml

```diff
@@ -185,7 +185,7 @@ jobs:
       run: |
         . ~/.graphscope_env
         python3 -m pip install libclang
-        git clone --single-branch -b v0.21.3 --depth=1 https://github.com/v6d-io/v6d.git /tmp/v6d
+        git clone --single-branch -b main --depth=1 https://github.com/v6d-io/v6d.git /tmp/v6d
```

Comment:
It would be better to fix to v0.21.5

Reply:
The main setting is just to test the latest vineyard, I will bump up to v0.22.0 if CI pass

Reply:
Updated to v0.22.0